source/la.html.erb

```diff
@@ -944,6 +944,9 @@
         <div class="bd">
           <h3>Rosemary Ledesma</h3>
           <div>
+            <a href="https://twitter.com/techgirlwonder" target="_blank">
```

Comment:
this is Hannah's twitter, not Rosemary's

Reply:
oh good catch!